mlrun/api/utils/clients/iguazio.py

```diff
@@ -505,11 +503,35 @@ def _generate_auth_info_from_session_verification_response(
         if x_unix_uid and x_unix_uid.lower() != "unknown":
             user_unix_id = int(x_unix_uid)
 
+        # get user and group ids from response body
+        response_body = response.json()
+        context_auth = get_in(
+            response_body, "data.attributes.context.authentication", {}
+        )
```

Comment:
Lets add a comment from which Iguazio version we are expecting to receive the `context_auth` in the response body.

Reply:
Also take a look at `if igz_version < semver.VersionInfo.parse` implementation in our code I think that can be suitable here as well.

Reply:
Also explain why the body response has precedence over the header response